screens/Home.js

```diff
@@ -317,6 +318,225 @@ class Home extends React.Component {
+      }, {
+        "gurmukhi": "Punhy mhlw 5",
+        "id": 17,
+        "translit": "funahe mahalaa panjavaa"
+      }, {
+        "gurmukhi": "cauboly",
+        "id": 18,
+        "translit": "chaubole"
+      }, {
+        "gurmukhi": "SsqR nwm mwlw",
+        "id": 19,
+        "translit": "shasatr naam maalaa"
+      }, {
+        "gurmukhi": "Awsw dI vwr",
+        "id": 90,
+        "translit": "aasaa dhee vaar"
+      }, {
+        "gurmukhi": "rhrwis swihb",
+        "id": 21,
+        "translit": "raharaas saahib"
+      }, {
+        "gurmukhi": "AwrqI",
+        "id": 22,
+        "translit": "aaratee"
+      }, {
+        "gurmukhi": "soihlw swihb",
+        "id": 23,
+        "translit": "sohilaa saahib"
+      }, {
+        "gurmukhi": "Ardws",
+        "id": 24,
+        "translit": "aradhaas"
+      }, {
+        "gurmukhi": "sRI BgauqI AsqoqR (pMQ pRkwS)",
+        "id": 25,
+        "translit": "sree bhagautee asatotr (pa(n)th prakaash)"
+      }, {
+        "gurmukhi": "sRI BgauqI AsqoqR (sRI hzUr swihb)",
+        "id": 26,
+        "translit": "sree bhagautee asatotr (sree hazoor saahib)"
+      }, {
+        "gurmukhi": "bwrh mwhw mWJ",
+        "id": 27,
+        "translit": "baareh maahaa maa(n)jh"
+      }, {
+        "gurmukhi": "bwrh mwhw svYXw",
+        "id": 28,
+        "translit": "baareh maahaa savaiyaa"
+      }, {
+        "gurmukhi": "Akwl ausqq",
+        "id": 29,
+        "translit": "akaal usatat"
+      }, {
+        "gurmukhi": "slok mhlw 9",
+        "id": 30,
+        "translit": "salok mahalaa nauvaa"
+      }, {
+        "gurmukhi": "suKmnI swihb",
+        "id": 31,
+        "translit": "sukhamanee saahib"
+      }, {
+        "gurmukhi": "suKmnw swihb",
+        "id": 32,
+        "translit": "sukhamanaa saahib"
+      }, {
+        "gurmukhi": "bwvn AKrI",
+        "id": 33,
+        "translit": "baavan akharee"
+      }, {
+        "gurmukhi": "isD gosit",
+        "id": 34,
+        "translit": "sidh gosaT"
+      }, {
+        "gurmukhi": "dKxI EAMkwru",
+        "id": 35,
+        "translit": "dhakhanee oa(n)kaaru"
+      }, {
+        "gurmukhi": "duK BMjnI swihb",
+        "id": 36,
+        "translit": "dhukh bha(n)janee saahib"
+      }, {
+        "folder": [Array],
+        "gurmukhi": "AMimRq bwxI",
+        "translit": "amirat baaNee"
+      }, {
+        "folder": [Array],
+        "gurmukhi": "Bgq bwxI",
+        "translit": "bhagat baaNee"
+      }, {
+        "folder": [Array],
+        "gurmukhi": "22 vwrW",
+        "translit": "22 varaa(n)"
+      }, {
+        "folder": [Array],
+        "gurmukhi": "sv`Xy",
+        "translit": "sava'ye"
+      }, {
+        "gurmukhi": "rwg mwlw",
+        "id": 38,
+        "translit": "raag maalaa"
+      }]
+    }
+}
+
+Home.propTypes={
+  appVersion:PropTypes.string.isRequired,
+  setAppVersion:PropTypes.func.isRequired,
+  baniLength:PropTypes.string.isRequired,
+  language:PropTypes.string.isRequired,
```

Comment:
veerji I believe the fomatter is still necessary for this project. The idea is to have standard code throughout the application. For instance, you have written  
```js
Home.propTypes={
  appVersion:PropTypes.string.isRequired,
  ...
}
```
Instead of the above, it should look something like
```js
Home.propTypes = {
  appVersion: PropTypes.string.isRequired,
  ...
}
```

Although it might seem minor, it makes the code much more readable and let it breathe. It will look much cleaner and easy to understand.

Reply:
Okay. Let me see why it's missed.

Reply:
This is fixed it's because our settings was overrided by prettierrc. so I removed the file now it's good.

Reply:
It looks good now.